### ISSUE TITLE
Diff size between "Show List" and "displayShow"

### ISSUE THREAD

**Issue by SRay77** [SickRage/sickrage-issues#596]
Hi,

```
Version:	Branch: master
Commit: 66fa37c556784fa3b41a06bb3043bd854d165c10
Version: 2016.01.20-1
User:	*****
Locale:	Language: fr_FR
Encoding: UTF-8
Configuration File:	/var/www/SiCKRAGE/config.ini
Database:	/var/www/SiCKRAGE/sickbeard.db
Cache Directory:	/var/www/SiCKRAGE/cache
Log Directory:	/var/www/SiCKRAGE/Logs
Arguments:	['-q', '--daemon', '--nolaunch', '--pidfile=/var/www/SiCKRAGE/sickrage.pid', '--datadir=/var/www/SiCKRAGE']
Python Version:	2.7.3 (default, Mar 13 2014, 11:03:55) [GCC 4.7.2]
```

There is a difference in size between the overall display of the series and displaying a series.
http://image.noelshack.com/fichiers/2016/03/1453475750-sr001.jpg
http://image.noelshack.com/fichiers/2016/03/1453475759-sr002.jpg
The right display is one of the series.
Is it possible to use only one method of calculation?

Thank's for fix :)

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
Fixed in develop

**Comment by SRay77** [comment]
Hi,

```
OS : Debian Wheezy 7.9
Linux thallium 3.2.0-4-amd64 #1 SMP Debian 3.2.68-1+deb7u6 x86_64 GNU/Linux

SickRage Version: 	Branch: master
Commit: 3f0efd1338faf8796fd355bada4f546ec1cb5ed6
Version: 2016.01.23-1
Python Version:	2.7.3 (default, Mar 13 2014, 11:03:55) [GCC 4.7.2]
SSL Version:	OpenSSL 1.0.1e 11 Feb 2013
Locale:	fr_FR.UTF-8
 	 
User:	*****
Program Directory:	/home/www/SiCKRAGE/SickRage
Configuration File:	/var/www/SiCKRAGE/config.ini
Database:	/var/www/SiCKRAGE/sickbeard.db
Cache Directory:	/var/www/SiCKRAGE/cache
Log Directory:	/var/www/SiCKRAGE/Logs
Arguments:	['-q', '--daemon', '--nolaunch', '--pidfile=/var/www/SiCKRAGE/sickrage.pid', '--datadir=/var/www/SiCKRAGE']
```

Unfortunately the problem is still present with the last update.

Sincerely,

**Comment by miigotu** [comment]
Yes, I had to revert the fix for this because it created problems on a specific python version and some ppl couldnt see their show page at all.

I will fix this another way.

**Comment by labrys** [comment]
@p0psicles Don't close things when pushing them to your own branch.

**Comment by miigotu** [comment]
@labrys that happens when you set your default branch to develop on your fork and rebase it =P

**Comment by labrys** [comment]
@miigotu I knew what happened just wanted him to be aware it was happening, which he didnt realize :)
